components/Button/button.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
I suggest adding meta author and description tag.

Reply:
Okay.

Reply:
Thanks!! Great suggestions:)